foodsaving/users/models.py

```diff
@@ -65,6 +66,7 @@ class User(AbstractBaseUser, BaseModel, LocationModel):
 
     deleted = BooleanField(default=False)
     deleted_at = DateTimeField(default=None, null=True)
+    current_group = ForeignKey(Group, blank=True, null=True)
```

Comment:
I tend to avoid imports just for foreign key references. You can use string reference there: https://docs.djangoproject.com/en/1.10/ref/models/fields/#django.db.models.ForeignKey